pkg/api/flow.go

```diff
@@ -1218,6 +1220,14 @@ func (h *flowHandler) initRoutes(r *mux.Router) {
 	//     "description": "successfully sent cloud event"
 	r.HandleFunc("/namespaces/{ns}/broadcast", h.BroadcastCloudevent).Name(RN_NamespaceEvent).Methods(http.MethodPost)
 
+	r.HandleFunc("/namespaces/{ns}/broadcast/{filter}", h.BroadcastCloudeventFilter).Name(RN_NamespaceEventFilter).Methods(http.MethodPost)
+
+	r.HandleFunc("/namespaces/{ns}/eventfilter/{filter}", h.CreateBroadcastCloudeventFilter).Name(RN_CreateNamespaceEventFilter).Methods(http.MethodPut)
+
+	r.HandleFunc("/namespaces/{ns}/eventfilter/{filter}", h.DeleteBroadcastCloudeventFilter).Name(RN_DeleteNamespaceEventFilter).Methods(http.MethodDelete)
+
+	r.HandleFunc("/namespaces/{ns}/eventfilter/update/{filter}", h.UpdateBroadcastCloudeventFilter).Name(RN_UpdateNamespaceEventFilter).Methods(http.MethodPut)
```

Comment:
When it is necessary to distinguish between CREATE and UPDATE operations, one canonical RESTful approach is to make the CREATE into a POST request and UPDATE into a PUT request. Another possible approach is to use either POST or PUT for CREATE, and then use PATCH for UPDATE. 

We have another option though. Sometimes, distinguishing between CREATE and UPDATE is not necessary. If the user PUTs data to a `{filter}`, we can check if it already exists, and if it doesn't then we CREATE it, but if it does then we UPDATE it. 

Either way, both CREATE and UPDATE should have the same PATH because they both reference the same thing. 

Either way,